src/libs/Navigation/NavigationRoot.js

```diff
@@ -66,7 +69,7 @@ class NavigationRoot extends Component {
                 onStateChange={this.parseAndStoreRoute}
                 onReady={this.props.onReady}
                 theme={navigationTheme}
-                ref={Navigation.navigationRef}
+                ref={navigationRef}
```

Comment:
Why this change?

Reply:
I made that change because I used `Navigation.setIsNavigationReady()` which I imported it here, and that import doesn't include the navigationRef export.
https://github.com/Expensify/App/blob/35b57776994b1f8efbdc099b55afe85bc6c388ab/src/libs/Navigation/NavigationRoot.js#L5

Is that ok or would you recommend doing something else?